feat: add --exec option (#25)
lightpohl/podcast-dl: README.md

| --reverse               |        | false    | Reverse download direction and start at last RSS item.                                                                                                        |
| --info                  |        | false    | Print retrieved podcast info instead of downloading.                                                                                                          |
| --list                  |        | false    | Print episode list instead of downloading.                                                                                                                    |

> | --exec                  | String | false    | Execute a command after files are downloaded.                                                                                                                 |

Reviewer: ```suggestion
| --exec                  | String | false    | Execute a command after each episode is downloaded                                                                                                                 |
```

- `podcast_title`: Title of the podcast feed.
- `podcast_link`: `link` value provided for the podcast feed. Typically the homepage URL.

> ## Executing process after downloading episode

Reviewer: ```suggestion
## Executing Process After Downloading Episode
```
